src/deprecated.jl

```diff
@@ -53,11 +66,12 @@ function rand_tangent(rng::AbstractRNG, x::T) where {T}
     end
     if all(tangent isa NoTangent for tangent in tangents)
         # if none of my fields can be perturbed then I can't be perturbed
+        depwarn_rt()
```

Comment:
should this be outside of the if statement so that both branches get it?

Reply:
No, the other branch calls `rand_tangent`, so will git it somewhere else.
Having it outside the if would result in it being printed multiple times